versions.json

```diff
@@ -302,5 +302,6 @@
     "0.36.0": "0.36.0",
     "0.35.0": "0.35.0",
     "0.34.0": "0.34.0"
-  }
+  },
+  "next": "2022-05-18"
```

Comment:
@meszarosdezso This is nicely implemented but I don't want to add more complications to [our release process](https://github.com/denoland/deno/blob/main/tools/cut_a_release.md). Maintaining this line is extra work for us. 

Thank you for the feature, but closing without merge.